book.toml

```diff
@@ -42,9 +42,15 @@ exclude = [
     "marketplace\\.visualstudio\\.com",
     "objects\\.githubusercontent\\.com",
     # The bug listing URL works only if an user is logged in, otherwise one gets 404.
-    "github\\.com/issues\\?",
+    "github\\.com/issues\\?q=.*",
+    # Similarly 500 is sometimes returned here.
+    "github\\.com/rust-lang/rust/pulls\\?q=.*",
```

Comment:
Question: is this still a problem? Occassionally github does have outages, but it seems surprising that basic queries like that can return 500.

Reply:
Either way, I don't think it really matters because this type of link should always be valid.